src/librustc/hir/map/mod.rs

```diff
@@ -501,10 +501,10 @@ impl<'hir> Map<'hir> {
     }
 
     /// Given a body owner's id, returns the `BodyId` associated with it.
-    pub fn body_owned_by(&self, id: NodeId) -> BodyId {
-        self.maybe_body_owned_by(id).unwrap_or_else(|| {
-            span_bug!(self.span(id), "body_owned_by: {} has no associated body",
-                      self.node_to_string(id));
+    pub fn body_owned_by(&self, id: HirId) -> BodyId {
+        self.maybe_body_owned_by_by_hir_id(id).unwrap_or_else(|| {
```

Comment:
Maybe rename `maybe_body_owned_by_by_hir_id` to `maybe_body_owned_by_hir_id`? =P

Reply:
Yeah, that one does sound silly ^^. I'm hoping to soon be able to just remove it in favor of just `maybe_body_owned_by`.